tools/cmake/README.md

```diff
@@ -82,27 +82,56 @@ If you're running CMake directly, you may need to pass it in yourself as follows
 cmake -S <source-dir> -B <build-dir> -DCMAKE_BUILD_TYPE=debug
 ```
 
-## How to build a greentea test
-
-Install prerequisites suggested in the previous section and follow the below steps to build:
-* Set your current directory to the test suite directory
-
-* Run the following command for the configuration CMake module to be generated
-  ```
-  mbedtools configure -t <TOOLCHAIN> -m <MBED_TARGET> --mbed-os-path /path/to/mbed-os
-  ```
-* Build the test binary with the full profile
+## How to build and run greentea tests
+
```

Comment:
I wonder if we need a note that migration of all tests is still in progress?

Reply:
This is the last remaining question. Can we merge this to get into the upcoming release (code freeze today).

Reply:
Should we wait until the migration of all tests is complete before merging this? I can see it causing confusion for users. This PR also removes the previous instructions for running greentea tests, which users could've been relying on. Rather than adding caveats or calling out both methods of running the tests, maybe we wait until the work to migrate all tests is complete, then the documentation is correct (and doesn't need to be changed again later)?